tests/reg_tests/test_adjoint.py

```diff
@@ -287,6 +362,12 @@ def cmplx_test_aero_dvs(self):
             # this will happen when training, but will hopefully be fixed down the line
             return
 
+        if self.name == "Rotating_wing":
+            rtol = 2e-6
```

Comment:
Any chance these tolerances can be tightened with the fix in #203?

Reply:
Unfortunately the tolerances need to be the same, when I tried smaller values I got the same relative error value as before